security/impersonating_user.rst

```diff
@@ -128,6 +136,10 @@ To switch back to the original user, use the special ``_exit`` username:
 
     http://example.com/somewhere?_switch_user=_exit
 
+.. tip::
+
+    You can leverage the Twig function ``impersonation_exit_path('/somewhere')``
+
```

Comment:
lets add the versionadded here too?

Reply:
not sure, as far as I know and can find on GitHub, impersonation_exit_path was added in v5